uberenv.py

```diff
@@ -530,9 +555,14 @@ def setup_paths_and_dirs(self):
         # Test if the override option was used (--spack-config-dir)
         self.spack_config_dir = self.opts["spack_config_dir"]
         if self.spack_config_dir is None:
+            # If command line option is not used, search for platform under
+            # given directory
             uberenv_plat = self.detect_platform()
             if not uberenv_plat is None:
```

Comment:
Another candidate for the `is not None` syntax (out of scope, but I can see it... ;) ).